packages/react-textarea/Spec.md

```diff
@@ -2,62 +2,174 @@
+
+It is usually used when the user will use more than one line of text, such as a review in a website.
+
+In some libraries, `TextArea` is a variant of `TextField` or `Input`.
 
 ## Prior Art
 
-_Include background research done for this component_
+| Component Library                                                                | Name                            |
+| -------------------------------------------------------------------------------- | ------------------------------- |
+| v8                                                                               | `TextField` variant `Multiline` |
+| v0                                                                               | `TextArea`                      |
+| [Ant Design](https://ant.design/components/input/)                               | `Input.TextArea`                |
+| [Material UI](https://v4.mui.com/components/text-fields/)                        | `TextField` variant `Multiline` |
+| [React Spectrum](https://react-spectrum.adobe.com/react-spectrum/TextArea.html)  | `TextArea`                      |
+| [React Bootstrap](https://react-bootstrap.github.io/forms/input-group/)          | `FormControl` as `textarea`     |
+| [Blueprint](https://blueprintjs.com/docs/#core/components/text-inputs.text-area) | `TextArea`                      |
+| [Grommet](https://v2.grommet.io/textarea)                                        | `TextArea`                      |
+
+- Most libraries have their own `TextArea` component while others make it a variant of another component.
+  - `React Bootstrap` uses as with `FormControl` as follows: `<FormControl as="textarea" />`
+  - `Material UI` and `v8` use the `TextField` control and added a variant `multiline` as follows: `<TextField multiline defaultValue="Default Value" />`
+  - `Ant Design` uses the `TextArea` subcomponent as follows: `<Input.TextArea placeholder="Text Area" />`
+
+#### TextArea in v8
 
-- _Link to Open UI research_
-- _Link to comparison of v7 and v0_
-- _Link to GitHub epic issue for the converged component_
+- `Textfield` component with `multiline` property. When the `multiline` property is set to true, it is rendered as a `textarea`.
+- Has descriptions, label, error message, and handles maximum character count.
+
+#### TextArea in v0
+
+- Doesn't have `label` prop.
+- Handles maximum character count.
 
 ## Sample Code
 
-_Provide some representative example code that uses the proposed API for the component_
+```tsx
+<TextArea
+  className="rootClassName"
+  style={{ background: 'blue' }}
+  id="textarea-1"
+  defaultValue="This is a TextArea"
+  onChange={(ev, data) => console.log(data.value)}
+  textArea={{ className: 'textAreaClassName', style: { color: 'green' } }}
+/>
+```
 
 ## Variants
 
-_Describe visual or functional variants of this control, if applicable. For example, a slider could have a 2D variant._
+- TextArea has six variants:
```

Comment:
I think it might be helpful to distinguish between appearance and behavior variants.